data/json/professions.json

```diff
@@ -7273,6 +7273,27 @@
       "female": { "entries": [ { "item": "panties" }, { "item": "bra" } ] }
     }
   },
+  {
+    "type": "profession",
+    "id": "muckracker",
+    "name": "Muckracker",
+    "description": "You've covered it all before, you know? War, Corruption, the bizzare and more, but even a pulitzer couldn't prepare you for this scoop.",
```

Comment:
```suggestion
    "description": "You've covered it all before, you know? War, corruption, the bizarre and more, but even a Pulitzer couldn't prepare you for this scoop.",
```
I've covered typos you know.
![dead-rising-frank-west](https://github.com/CleverRaven/Cataclysm-DDA/assets/99621099/a77f3275-5c4e-42ea-9137-cb8a3bf734b1)